api/data_explorer/dataset_faceted_search.py

```diff
@@ -1,40 +1,78 @@
 """Subclass of FacetedSearch for Data explorer datasets."""
 
+import csv
 import jsmin
 import json
 
+from flask import current_app
+
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import HistogramFacet
 from elasticsearch_dsl import FacetedSearch
+from elasticsearch_dsl import Mapping
 from elasticsearch_dsl import TermsFacet
 
 
-def _get_index_name():
-    """Gets index name from /app/dataset.json."""
-    with open('/app/dataset.json') as f:
+def get_index_name():
+    """Gets index name from dataset.json."""
+    with open('/app/config/dataset.json') as f:
         # Remove comments using jsmin, as recommended by JSON creator
         # (https://plus.google.com/+DouglasCrockfordEsq/posts/RK8qyGVaGSr).
         dataset = json.loads(jsmin.jsmin(f.read()))
         return dataset['name']
 
 
+def get_facets():
```

Comment:
I'm not too familiar with how we're structuring this area of the code base, is this function intended to replace the one in facets_controller? If so, can we either delete that or yank the get_facets logic out of here and extract it into the facets_controller, passing in this instance to prevent duplicate instantiations of this class?